Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -138,6 +138,11 @@ struct VideoConfig final
   int iSaveTargetId = 0;  // TODO: Should be dropped
   u32 iMissingColorValue = 0;
   bool bFastTextureSampling = false;
+  bool bEFBBloomFixDownscale = false;
+  unsigned int iEFBBloomFixWidth = 0;
+  bool bEFBBloomFixAltFilter = false;
+  bool bEFBBloomFixSkipFirst = false;
+  bool bEFBBloomFixEnabled = false;
```

Comment:
Is there a reason why in some places the "enable" option is listed last, like in here or the ini's but others like the GUI, list it first to be logical

Reply:
Good point